arches_controlled_lists/models.py

```diff
@@ -372,3 +415,6 @@ class NodeProxy(Node):
 
     class Meta:
         proxy = True
+
+
+from arches_controlled_lists.datatypes.datatypes import ReferenceLabel
```

Comment:
why import at the bottom of the file?

Reply:
It's to hide a circular import (that's only needed for the type annotation on L239). We're a long way from activating any python type checkers (given that we're only hapharzadly adding annotations when we think it aids comprehension), but the yellow squiggly in VS Code sure is ugly. I'm happy to leave the yellow squiggle in if you'd prefer.

Reply:
ya if possible, could this be any other way? Feels like an anti-pattern to import at the bottom of a file to get around a circular dep.

Reply:
Wait, I can probably just do the standard `if TYPE_CHECKING` shuffle, taking a look

Reply:
That did it!